tasks/Add-music.md

```diff
@@ -13,6 +13,12 @@ Requirements:
 - It should be music relevant to the theme, try not to add any songs
 - Please make sure to use the existing API.
 - The music should be your own work. Please make sure you have the rights to any music that you add.
+- the music of any tile stops playing when the player leaves not moves
```

Comment:
This should be done by the API but it is good to add, we should refer to the API. The documentation is merged.